### ISSUE TITLE
Config to enable/disable different animations?

### ISSUE THREAD

**Issue by PPMDJ** [Steveplays28/player-animation-rework#6]
In case the player would prefer to turn off specific animations, or would prefer certain animations from other mods, having a config file is super important! 

Also, having config options for specific parts of the animations would be awesome too. If the player wants to slow down/speed up certain animations or toggle other things about the animations, having a detailed config file is a necessity. At least have toggleable animations tho.

**Comment by Steveplays28** [comment]
Good idea, I'll see if I can add support for resourcepack animation loading too.
